go/example.go

```diff
@@ -16,11 +16,12 @@ import (
 	"context"
 	"crypto/tls"
 	"fmt"
+	"github.com/apache/arrow-go/v18/arrow/flight"
```

Comment:
checkstyle

imports should be separated for two groups, standards and other

Reply:
@IsupovEvgenii let's create a ticket about doing the CI in this repo?